modules/apps/site-initializer/site-initializer-liferay-learn/src/main/resources/site-initializer/fragments/group/public-sites-navigation/fragments/public-sites-navigation/index.js

```diff
@@ -11,3 +11,31 @@
  * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
  * details.
  */
+
+const menuButtonGroup = document.querySelector('.menu-button-group');
```

Comment:
@liaoethan  These selectors are pretty non-specific to use on the whole document. You can instead just add `/* eslint-disable no-undef */` to ignore the fragmentElement error